.env.example

```diff
@@ -1,4 +1,6 @@
-APP_PORT=
+APP_PORT=3000
```

Comment:
Rename app port to http port and also add `http addr`. and set 127.0.0.1 by default